src/app/games/siam/SiamTutorial.ts

```diff
@@ -24,10 +24,11 @@ export class SiamTutorial {
             $localize`Goal of the game`,
             $localize`The goal at Siam is to be the first to push a mountain out of the board.
         The initial board contains three mountains, and no pieces are initially on the board.
-        During its turn, a player can do one of the following three actions:
+        During its turn, a player can do one of the following actions:
```

Comment:
1. "As long as you do not have 5 pieces on the board" p-e maintenant qu'on voit les remaining piece cette phrase est sub-optimale
2. "your piece" but which are theeeeey, am I dark or light ? Oui, on devrais préciser ça dans tout les jeux j'en ai peur :D
3. "alongside the board" c'est le meilleurs moyen de dire en haut et en bas ou de part et d'autres ?
4. "Select the orientation of your piece by clickin gone one of the arrows that appear on top of the board." what does clickin' gone one mean :D ?
5. "Try to move the piece that is on the board one square upwards and to orient it downwards." p-e c'est plus malin d'apprendre à décaler à droite et orienter à gauche, haha :D
6. a.The number of the pieces (opponent's or not) that are facing yours (called the resistants) must be less than the number of pieces that are oriented in the same direction as the push, yourself included (the pushers).
b. The number of mountains on that line must be less or equal to the difference between the pushers and the resistants.
Ou plutôt, plus simple:
The number of resistant+mountain must be <= if it contains mountain, strictly < if it does not ?
8. Last step: you have *BOTH THOSES* way to finish the game ?

Multi-comment!

Reply:
tout adapté / reformulé !

Reply:
Chaque montagne supplémentaire après la première requiert un pousseur en plus alors ? T'as une formulation claire ET concise pour pas tuer le lecteur ? Car la je comprends pas :D

Reply:
ça a été reformulé après discussion donc !